nix/hix.nix

```diff
@@ -0,0 +1,27 @@
+{pkgs, config, ...}: {
+  # name = "project-name";
+  compiler-nix-name = "ghc8107"; # Version of GHC to use
+
+  # Cross compilation support:
+  # crossPlatforms = p: pkgs.lib.optionals pkgs.stdenv.hostPlatform.isx86_64 ([
+  #   p.mingwW64
+  #   p.ghcjs
+  # ] ++ pkgs.lib.optionals pkgs.stdenv.hostPlatform.isLinux [
+  #   p.musl64
+  # ]);
+
+  # Tools to include in the development shell
+  shell = {
+    tools = {
+      cabal = "latest";
```

Comment:
It might be a good idea to use the same cabal version as the CI system.

GitHub Actions is using cabal version 3.8:

https://github.com/geniusyield/atlas/blob/9e29522b1fc6f7193929fd1069db1bc1dbe446e5/.github/workflows/haskell.yml#L55